pkg/reconciler/pipeline/controller.go

```diff
@@ -144,7 +149,11 @@ func (r *pipelineReconciler) reconcile(ctx context.Context, pl *dfv1.Pipeline) (
 	}
 
```

Comment:
L143, I think it might also worth a normal event. `log.Infow("Updated pipeline phase", zap.String("originalPhase", string(oldPhase)), zap.String("currentPhase", string(pl.Status.Phase)))`

Reply:
Should this also include oldPhase to currentPhase in the message?
```
r.recorder.Event(pl, corev1.EventTypeNormal, "UpdatePipelinePhase", "Updated pipeline phase")
```

Reply:
I think yes. It makes the event more detailed and easy for debugging.